lib/membrane_rtc_engine/engine.ex

```diff
@@ -163,7 +234,8 @@ defmodule Membrane.RTC.Engine do
   Endpoint configuration options.
 
   * `peer_id` - associate endpoint with exisiting peer
-  * `endpoint_id` - assign endpoint id. If not provided it will be generated by RTC Engine. This option cannot be used together with `peer_id`. Endpoints associated with peers have the id `peer_id`.
+  * `endpoint_id` - assign endpoint id. If not provided it will be generated by RTC Engine. This option cannot be used together with `peer_id`.
+  Endpoints associated with peers have the id `peer_id`.
   * `node` - node on which endpoint
   """
   @type endpoint_options_t() :: [
```

Comment:
During my last work with Engine, I found out that the current implementation of handlers for `:add_endpoint`,  require a parameter node in options. So we can add this to docs that node option is required or change the handlers.

Reply:
I have refactored a little bit code so now it's optional